features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -1,7 +1,9 @@
 <%
 log = new Log(),
 apiProvider = jagg.module("manager").getAPIProviderObj(),
+usageClient = jagg.module("manager").getUsageClient(),
 user = jagg.getUser();
+var gson=com.google.gson.Gson();
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea